src/NUnitFramework/framework/Api/DefaultTestAssemblyBuilder.cs

```diff
@@ -148,22 +153,24 @@ private TestSuite Build(Assembly assembly, string assemblyPath, IDictionary<stri
                             }
                         }
                 }
+                
+                var fixtures = GetFixtures(assembly, GetValueOrDefault(options, FrameworkPackageSettings.LOAD) as IList);
```

Comment:
Consistency, instead of doing the same thing multiple ways. No problem to change back.

Perhaps also I should have copied the original `ContainsKey`...indexer pattern. Writing that code is harder for me because it's not only less efficient, it's less DRY, but I can see how maybe you'd want to discuss that first.